azure_iot_edge/assets/configuration/spec.yaml

```diff
@@ -4,7 +4,7 @@ files:
     options:
       - template: init_config
         options:
-          - template: init_config/default
+          - template: init_config/openmetrics
```

Comment:
Why not keep `init_config/default`?

Reply:
Ah nevermind, I see that `init_config/openmetrics` already contains `init_config/default`